src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -1448,7 +1449,8 @@ private ConnectorParameterInternals Initialize()
 
                                             bodyPropertyHiddenRequired = ConnectorSettings.Compatibility != ConnectorCompatibility.PowerAppsCompatibility || !requestBody.Required;
                                         }
-                                        else if (ConnectorSettings.Compatibility == ConnectorCompatibility.SwaggerCompatibility)
+                                        else if (ConnectorSettings.Compatibility == ConnectorCompatibility.SwaggerCompatibility || 
+                                                 ConnectorSettings.Compatibility == ConnectorCompatibility.CdpCompatibility)
```

Comment:
Can we use an extension method instead? 

So we don't need each caller to make this decision. 



ConnectorSettings.Compatibility.IsXXXX()

and 

public static IsXXXX(this ConnectorCompatibility settings)
{
return settings == ConnectorCompatibility.SwaggerCompatibility ||      
                                            settings == ConnectorCompatibility.CdpCompatibility);

}